internal/status/status.go

```diff
@@ -100,11 +100,6 @@ func (s *Status) Err() error {
 	return (*Error)(s.Proto())
 }
 
-func (s *Status) Error() string {
```

Comment:
Would it make more sense to return an empty string for an `OK` status instead of totally getting rid of this method?

Reply:
No, it's important that this does not satisfy the `error` interface.  A non-nil error is considered "bad" but a non-nil `*Status` can be "OK".

Reply:
Ack.